wiki/Glossary/en.md

```diff
@@ -486,21 +487,21 @@ Upon the creation of the marathon beatmap, one should name their beatmap version
 
 ### MAT
 
-**As of the 23. April 2013, all members of the MAT at that time were promoted as full members of the [BAT](/wiki/BAT), with the MAT team decommissioned.**
+**As of the 2013-04-23, all members of the MAT at that time were promoted as full members of the [BAT](/wiki/BAT), with the MAT team decommissioned.**
 
-The MAT, meaning _Modding Assistance Team_, was a group of users experienced in modding. MAT members were able to mark any beatmap they approve of with a bubble. This bubble would note that the beatmap as worthy of further consideration by the BAT.
+*Short for "Modding Assistance Team".*
+
+A group of users experienced in modding. MAT members were able to mark any beatmap they approve of with a bubble. This bubble would note that the beatmap as worthy of further consideration by the BAT.
 
 ### Mod Request
 
-A mod request is when a mapper is in need of help.
-This can be from fixing possible mistakes to requesting for [guest difficulties](#guest-difficulty).
-While, typically, the song is still playable, the only reason someone would scroll here is if (and only if) you know what you are (and will be) doing and willing to mod the map for them.
+A term used when a mapper is in need of help. This can be from fixing possible mistakes to requesting for [guest difficulties](#guest-difficulty). While, typically, the song is still playable, the only reason someone would scroll here is if (and only if) you know what you are (and will be) doing and willing to mod the map for them.
```

Comment:
```suggestion
A term used when a creator is in need of help. This can be from fixing possible mistakes to requesting for [guest difficulties](#guest-difficulty). While, typically, the song is still playable, the only reason someone would scroll here is if (and only if) you know what you are (and will be) doing and willing to mod the map for them.
```